plugins/catalog/src/components/CatalogTable/CatalogTable.tsx

```diff
@@ -83,8 +88,52 @@ export const CatalogTable: FC<CatalogTableProps> = ({
     );
   }
 
+  const actions: TableProps<Entity>['actions'] = [
+    (rowData: Entity) => {
+      const location = findLocationForEntityMeta(rowData.metadata);
+      return {
+        icon: GitHub,
+        tooltip: 'View on GitHub',
+        onClick: () => {
+          if (!location) return;
+          window.open(location.target, '_blank');
+        },
+        hidden: location?.type !== 'github',
+      };
+    },
+    (rowData: Entity) => {
+      const createEditLink = (location: LocationSpec): string => {
+        switch (location.type) {
+          case 'github':
+            return location.target.replace('/blob/', '/edit/');
+          default:
+            return location.target;
+        }
+      };
+      const location = findLocationForEntityMeta(rowData.metadata);
+      return {
+        icon: Edit,
+        tooltip: 'Edit',
+        // iconProps: { size: 'small' },
```

Comment:
Can we remove this?

Reply:
Ah I forgot about this line. The reason it's commented out is that the types refused to accept it being there. I'll double check that the visuals are OK without it and then remove entirely if possible, otherwise try to make it compile.

Reply:
Replaced with actual components and using the right prop :)